backend/form-flow/src/main/kotlin/com/ritense/formflow/FormFlowProcessLinkActivityHandler.kt

```
@@ -122,12 +123,23 @@ class FormFlowProcessLinkActivityHandler(
-
-        val formFlowDefinition = formFlowService
-            .findDefinition(processLink.formFlowDefinitionKey, processDefinitionCaseDefinitionLink.id.caseDefinitionId)!!
+        val formFlowDefinition = findFormFlowDefinition(processLink)
         return formFlowService.save(formFlowDefinition.createInstance(additionalProperties))
     }
 
+    private fun findFormFlowDefinition(processLink: FormFlowProcessLink): FormFlowDefinition {
+        val caseLink = try {
+            processDefinitionCaseDefinitionService
+                .findByProcessDefinitionId(ProcessDefinitionId(processLink.processDefinitionId))
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)